### PR TITLE
CLI "pyrsia status" total artifacts explanation

### LINKED ISSUE #555: CLI "artifacts count" expansion
Artifacts counts should be broken down to manifest + blobs count, CLI should display :
Artifacts Count: 3 (Blobs: 2, Manifests:1)

### PR DESCRIPTION
**Pull request by MitaliBo**
## Description

Fixes #555 and #506

This PR explains the total number of artifacts shown by "pyrsia status" command by dividing the count into manifests and blobs. Please refer the screenshot attached.

Screen Shot 2022-05-09 at 12 13 34 AM

## PR Checklist

- [x] I've read the [contributing guidelines](https://github.com/pyrsia/.github/blob/main/contributing.md).
- [x] I've read ["What is a Good PR?"](https://github.com/pyrsia/pyrsia/blob/main/docs/good_pr.md)
- [x] I've included a good title and brief description along with how to review them.
- [x] I've linked any associated an [issue](https://github.com/pyrsia/pyrsia/issues).
- [x] I've requested a review  from `pyrsia/collaborators`.

### Code Contributions

- [x] I've built the code `cargo build --all-targets` successfully.
- [x] I've run the unit tests `cargo test --workspace` and everything passes.
- [x] I've made sure my rust toolchain is current `rustup update`.

### CODE REVIEW

**Review by prince-chrismc — DISMISSED** [review]

File: docs/tutorials/demo.md [hunk 1]
```diff
@@ -199,9 +199,9 @@ pyrsia -s
 You should see something very similar to:
 ```
 Connected Peers Count:       1
-Artifacts Count:             0
-Total Disk Space Allocated:  10.84 GB
-Disk Space Used:             0.0000%
+Artifacts Count:             2 {"manifests": 1, "blobs": 2}
```
Comment (fishseabowl) [inline docs/tutorials/demo.md:202]: For me there is some confusion about artifact counts, manifests and blobs.

Comment (MitaliBo) [reply docs/tutorials/demo.md:202]: what is the confusion?

Comment (fishseabowl) [reply docs/tutorials/demo.md:202]: For issue #555 : 
```
  Artifacts counts should be broken down to manifest + blobs count, CLI should display :
  Artifacts Count: 3 (Blobs: 2, Manifests:1)
```

But in the above Artifacts counts is not equal to manifest + blobs count.

Comment (fishseabowl) [reply docs/tutorials/demo.md:202]: In my understanding, an artifact is a complete environment with all tools, dependencies, etc. A manifest is metadata about an image, such as layers, sizes, and digest. Blobs are docker layer data. So I guess an artifact should consist of a manifest and some blobs. Please correct me if I misunderstood.

Comment (MitaliBo) [reply docs/tutorials/demo.md:202]: if you are looking for documentation that is just typo, its actually sum of blobs and manifest only, please refer my screenshot above for code validation

**Comment by prince-chrismc** [comment]
Sorry about the merge conflicts 🙊

File: docs/tutorials/demo.md (hunk 1 above)
Comment (prince-chrismc) [reply docs/tutorials/demo.md]: Good catch! Good docs make a huge difference!

**Review by prince-chrismc — APPROVED** [review]

**Comment by MitaliBo** [comment]
> Sorry about the merge conflicts 🙊

oh i missed the train :(

**Pull request merged**